c-dependencies/js-compute-runtime/js-compute-builtins.cpp

```diff
@@ -5451,6 +5474,437 @@ bool constructor(JSContext *cx, unsigned argc, Value *vp) {
 }
 } // namespace TextDecoder
 
+#ifdef ENABLE_OBJECT_STORE
```

Comment:
The implementation is currently behind a flag because viceroy does not yet support the required hostcalls. If we compiled with the ObjectStore implementation then when viceroy tries to run the application it will crash due to hostcalls being called which don't exist.